server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java

```diff
@@ -23,6 +23,7 @@
 import static com.jayway.restassured.RestAssured.with;
```

Comment:
I'd like to see an other test here : what happens when you save a message via setMessages to a mailbox that is neither outbox nor draft.

Is it accepted ? Rejected ?